src/client.ts

```diff
@@ -17,25 +17,17 @@ import { addUrlQuery, defaultPortalUrl, makeUrl } from "./utils";
 import { getFile, setFile } from "./skydb";
 import { lookupRegistry, updateRegistry } from "./registry";
 
+export type CustomClientOptions = {
+  APIKey?: string; // authentication password to use
```

Comment:
Can we use JSDoc to document these parameters?

Reply:
jsdoc doesn't apply to typescript

Reply:
replaced `// comment` with `/** comment */` so they are preserved in the resulting d.ts file